src/libraries/Microsoft.Extensions.Logging.Console/tests/Microsoft.Extensions.Logging.Console.Tests/ConsoleLoggerExtensionsTests.cs

```diff
@@ -374,8 +374,7 @@ public void AddConsole_MaxQueueLengthSetToNegativeOrZero_Throws(int invalidMaxQu
                 )
                 .BuildServiceProvider();
 
-            // the configuration binder throws TargetInvocationException when setting options property MaxQueueLength throws exception
-            Assert.Throws<TargetInvocationException>(() => serviceProvider.GetRequiredService<ILoggerProvider>());
+            Assert.Throws<ArgumentOutOfRangeException>(() => serviceProvider.GetRequiredService<ILoggerProvider>());
```

Comment:
>ArgumentOutOfRangeException [](http://example.com/codeflow?start=26&length=27)

should we keep the old exception in such case?

Reply:
`TargetInvocationException` shouldn't have been thrown in the first place. Maybe that is a bug we should log on ConfigurationBinder. We should be passing `BindingFlags.DoNotWrapExceptions`.

Reply:
I was only asking if we should be concerned about app compatibility. I think it is very unlikely to have anyone depend on such thrown exception though.